src/main/kotlin/gg/flyte/twilight/scoreboard/TwilightScoreboard.kt

```diff
@@ -0,0 +1,156 @@
+package gg.flyte.twilight.scoreboard
+
+import net.kyori.adventure.text.Component
+import org.bukkit.Bukkit
+import org.bukkit.entity.Player
+import org.bukkit.plugin.java.JavaPlugin
+import org.bukkit.scheduler.BukkitRunnable
+import org.bukkit.scheduler.BukkitTask
+import org.bukkit.scoreboard.DisplaySlot
+import org.bukkit.scoreboard.Scoreboard
+
+class TwilightScoreboard(private val plugin: JavaPlugin) {
+    private val scoreboard: Scoreboard = Bukkit.getScoreboardManager().newScoreboard
+    private var objective = scoreboard.registerNewObjective("sidebar", "dummy")
```

Comment:
This objective name should probably be "twilight-sidebar" or something so we can easily obtain all Twilight related ones.